client/.gitignore

```diff
@@ -1,2 +0,0 @@
-node_modules
```

Comment:
Ok, I missed that sorry. I'd still like to remove the mailers until it becomes a feature. Avoid having unused coded.